cxx-sensors/src/main/java/org/sonar/cxx/sensors/compiler/gcc/CxxCompilerGccSensor.java

```diff
@@ -32,8 +32,9 @@
   public static final String REPORT_REGEX_DEF = "gcc.regex";
   public static final String REPORT_CHARSET_DEF = "gcc.charset";
   public static final String DEFAULT_CHARSET_DEF = "UTF-8";
+  public static final String DEFAULT_ID = "enabled by default";
```

Comment:
Normal message keys should have no blanks. Maybe we should change it in XML file? Just default. 
Is Code Smell / Critical / 5min the right default?

Reply:
For now all the gcc rules have the same type/severity/remediation effort.
I agree this should be questioned, but I suggest to address that in another PR.

At first I thought I would add a new rule, but then I found this one *enabled by default*, which does not correspond to any matching gcc warning id (all other id looks like `-Wsomething` -- i.e. the gcc activation parameter).
If you think it's better to change the id, I'm fine.